pkg/api/arc.yaml

```diff
@@ -1078,6 +1080,13 @@ components:
         Whether we should perform cumulative fee validation for fee consolidation txs or not.
       schema:
         type: boolean
+    strictCumulativeFeeValidation:
+      name: X-StrictCumulativeFeeValidation
+      in: header
+      description: >-
+        Whether we should perform cumulative fee validation for fee consolidation txs or not with unmined ancestors limit.
```

Comment:
Whether we should perform cumulative fee validation for fee consolidation txs with unmined ancestors limit or not.

Reply:
Changed